frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -622,11 +658,19 @@ pub mod pallet {
 				Phase::Signed | Phase::Off
 					if remaining <= unsigned_deadline && remaining > Zero::zero() =>
 				{
-					// determine if followed by signed or not.
+					// our needs vary according to whether or not the unsigned phase follows a signed phase
 					let (need_snapshot, enabled, signed_weight) = if current_phase == Phase::Signed {
-						// followed by a signed phase: close the signed phase, no need for snapshot.
-						// TODO: proper weight https://github.com/paritytech/substrate/pull/7910.
-						(false, true, Weight::zero())
+						// there was previously a signed phase: close the signed phase, no need for snapshot.
+						//
+						// Notes:
+						//
+						//   - If the signed phase produced a viable solution, we disable the unsigned
```

Comment:
not sure if this is the case anymore. But keep an eye on this, and maybe make it configurable.

Reply:
given the past events I'd say we want to open the unsigned phase all the time for now.

Reply:
ea2b1abd2